Core/include/Acts/EventData/Measurement.hpp

```diff
@@ -258,21 +277,27 @@ class Measurement {
   /// @return @c true if both objects are not equal, otherwise @c false
   ///
   /// @sa Measurement::operator==
-  bool operator!=(const Measurement<source_link_t, params...>& rhs) const {
+  bool
+  operator!=(const Measurement<source_link_t, params...>& rhs) const {
     return !(*this == rhs);
   }
 
   /// @projection operator
-  static Projection projector() { return ParSet_t::projector(); }
+  static Projection
+  projector() {
+    return ParSet_t::projector();
+  }
 
-  friend std::ostream& operator<<(
-      std::ostream& out, const Measurement<source_link_t, params...>& m) {
+  friend std::ostream&
+  operator<<(std::ostream& out,
+             const Measurement<source_link_t, params...>& m) {
```

Comment:
This alignment is the worst. It results in every single change to anything causes a huge multiline diff, and in my opinion this complicates PR reviewing, as the Github diff viewer can't make sense out of it anymore.

Reply:
I think it's rather unlikely 10 arguments fit in one line, regardless how short their names are. Also you misspelled "this alignment is the best".
If I scan a line for arguments with my eyes, I try to parse commas. in this very line it would have taken me an extra second to see that it's actually only two arguments, not 3. With the new alignment there's no question about that.

Reply:
> Worst case example: change one argument name -> 10 line diff.

This is one of the main reasons I do not like this either.

Reply:
This argument is vastly exaggerated though. with this change, arguments only stay in one line if they all fit. if you have three arguments and you add a fourth one, chances are you already pass the limit, which would lead to 4 lines changed, not 10. and how often does that happen, and even if it does happen, the newly aligned arguments are more readable